app/src/main/java/org/sopt/dosopttemplate/util/IntentExt.kt

```diff
@@ -2,11 +2,22 @@ package org.sopt.dosopttemplate.util
 
 import android.content.Intent
 import android.os.Build
+import android.util.Log
+import org.sopt.dosopttemplate.data.Profile
 
 fun <T> Intent.getParcelable(name: String, clazz: Class<T>): T? {
     return if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
         getParcelableExtra(name, clazz)
     }
     else
         getParcelableExtra(name)
+}
+
+fun logProfile(profile: Profile) {
+    Log.v("nick name", profile.name.toString())
+    Log.v("mbti", profile.MBTI.toString())
+    Log.v("music title", profile.getMusic())
+    Log.v("intro", profile.intro.toString())
+
+
```

Comment:
```suggestion
fun logProfile(profile: Profile) {
    Log.v("profile", profile.toString())
```

profile 같은 data 클래스는 toString을 상속받아 내부에서 기능을 구현해두었기 때문에 모든 프로퍼티가 Log내에 찍힐것입니다!

Reply:
의진님이 잘 설명해주셨는데요, data class는 자바에는 없는 코틀린의 특별한 class이기 때문에 공부해보셔도 좋을 듯 합니다!

Reply:
오호랏 감사합니다!